src/SearchResults.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import FakeBookings from "./data/fakeBookings.json";
```

Comment:
You should only capitalize the first letter of the variable if it is:
- A React component
- A class

In this case, would be better if you called it fakeBookings.